etl/steps/data/garden/animal_welfare/2023-08-15/number_of_farmed_decapod_crustaceans.meta.yml

```diff
@@ -0,0 +1,30 @@
+definitions:
+  common: |
+    This study estimates the annual number of farmed decapod crustaceans slaughtered for food using data on aquaculture production tonnages (from the Food and Agriculture Organization of the United Nations) and mean weights estimated from available decapod crustaceans size data (from a range of internet sources). For species categories where no estimated mean weight was obtained, a generic mean weight was used to estimate numbers.
```

Comment:
Maybe it's better to specify what decapod crustaceans are https://en.wikipedia.org/wiki/Decapoda